app/frontend/courses/report/components/CoursesReport__Overview.res

```diff
@@ -125,7 +125,7 @@ let milestoneTargetsCompletionStatus = overview => {
       {(completedMilestoneTargets->string_of_int ++ " / " ++ totalMilestoneTargets->string_of_int)
         ->str}
       <span className="px-2 text-gray-300"> {"|"->str} </span>
-      {milestoneTargetCompletionPercentage ++ "% completed" |> str}
+      {milestoneTargetCompletionPercentage ++ t("percentage_completed") |> str}
```

Comment:
Let the user decide how they want to show the number and percentage; Pass the `milestoneTargetCompletionPercentage` value into`en.yml`